ee/ucp/authorization/isolate-nodes.md

```diff
@@ -226,20 +226,42 @@ Select the **ops-nodes** namespace, and create a `Full Control` grant for the
 
 ### Link the namespace to a node collection
```

Comment:
With UCP 3.1, linking the  namespace to a node collection is not available. It's obsolete.

Reply:
Also, I don't think the title of the section should be `Deploy a Kubernetes application` It's misleading because that's not really what we're trying to do here. The goal should be around `Isolating Nodes to Kubernetes Namespaces` then you can start with the first step which is creating a namespace.

Reply:
I think I agree with "I'm not too sure if it's good to have a single file describing Node RBAC for Swarm and Kubernetes". As the Kubernetes node isolation solution here doesn't integrated with Collections, which is primarily what this page describes.